design-tokens/src/formats/cssBreakpoints.ts

```diff
@@ -1,20 +1,21 @@
-import StyleDictionary from 'style-dictionary';
-import { FormatterArguments } from 'style-dictionary/types/Format.js';
+import { FormatFn, FormatFnArguments } from 'style-dictionary/types';
+import { fileHeader, formattedVariables } from 'style-dictionary/utils';
 
-export const cssBreakpoints = ({
+export const cssBreakpoints: FormatFn = async ({
   dictionary,
   file,
   options,
-}: FormatterArguments) => {
+}: FormatFnArguments) => {
   const { outputReferences, mediaQuery } = options;
-  let output = `:root {\n${StyleDictionary.formatHelpers.formattedVariables({
+  let output = `:root {\n${formattedVariables({
     format: 'css',
     dictionary,
     outputReferences,
+    usesDtcg: true,
```

Comment:
What does `usesDtcg` represent?

Reply:
It's a way to say we are using [Design Token Community Group](https://tr.designtokens.org/format/) syntax of $value